lib/child_process.js

```diff
@@ -50,8 +50,8 @@ exports.fork = function(modulePath /*, args, options*/) {
   args = execArgv.concat([modulePath], args);
 
   if (!Array.isArray(options.stdio)) {
-    // Leave stdin open for the IPC channel. stdout and stderr should be the
-    // same as the parent's if silent isn't set.
+    // Use a separate fd=4 for IPC channel. Inherit stdin, stdout and stderr 
```

Comment:
Please leave the "the" before "IPC channel". Also, please add a comma after "stdout."